docs/rst/conf.py

```diff
@@ -12,34 +13,26 @@
 #
 # All configuration values have a default value; values that are commented out
 # serve to show the default value.
-"""Documentation Configuration."""
 
-import sys
-import os
+import datetime
 
-# pip install sphinx_rtd_theme
-# import sphinx_rtd_theme
-# html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+# pip install sphinx_ansible_theme
```

Comment:
I don't think this is useful since all the supported configs have this info in the right place
```suggestion
```